src/sections/Company/News-grid/NewsGrid.style.js

```diff
@@ -6,10 +6,12 @@ export const NewsPageWrapper = styled.div`
     }
     .page-header{
         margin:5rem auto;
+        margin-top: 1rem;
```

Comment:
Aggregate into one line, `margin: 1rem auto 5rem`

Reply:
I have made the necessary changes